lib/api/api.go

```diff
@@ -81,7 +81,6 @@ type service struct {
 	fss                  model.FolderSummaryService
 	urService            *ur.Service
 	systemConfigMut      sync.Mutex // serializes posts to /rest/system/config
-	cpu                  Rater
```

Comment:
Is this Rater interface used anywhere else? Otherwise kill it

Reply:
Good catch!